src/test/java/net/minecraftforge/debug/EnumPlantTypeTest.java

```diff
@@ -27,11 +33,15 @@ public void onInit(FMLInitializationEvent event)
         finally
         {
             if (biomeType == null || !biomeType.name().equals("FAKE"))
+            {
                 LOGGER.warn("EnumHelper in BiomeType is working incorrectly!");
+            }
         }
         EnumPlantType plantType = null;
         if (plantType == null || !plantType.name().equals("FAKE"))
+        {
             ;
+        }
```

Comment:
Why do you even keep this block when it does not have an `else` statement after?